app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -54,6 +55,15 @@ class PatientSummaryScreenControllerTest {
 
   private val uiEvents = PublishSubject.create<UiEvent>()
   private val reporter = MockReporter()
+
+  // If the placeholder count is changed, the `params for placeholder bp items`
+  // method will also need to be changed to account for the change in the
+  // number of placeholder.
+
+  // TODO: Change the method to generate test inputs dynamically based on
+  // the number of placeholders declared.
```

Comment:
@vinaysshenoy you should try making it a habit to use multi-line comments for code that are outside functions. This way, you can also add a link to `params for placeholder bp items()` function. I'll do this for you.

Reply:
@vinaysshenoy you should try using multi-line comments for code that are outside functions, as per the standard. This way, you can also add a link to `params for placeholder bp items()` function. I'll do this for you.